server/bin/unittests

```diff
@@ -17,13 +20,31 @@ export _tdir=$(dirname $(readlink -f $0))
 export _gitdir=$(dirname $(dirname $_tdir))
 export _tlib=${_tdir}/../lib
 export _testbase=/var/tmp/pbench-test-server
+export _testvenv=${_testbase}/venv
+
 rm -rf ${_testbase}
 mkdir ${_testbase}
 if [[ ! -d $_testbase ]]; then
     echo "ERROR: failed to create test base directory, \"$_testbase\"" >&2
     exit 1
 fi
 
+function _install_venv {
```

Comment:
And it seems that the fact that this function is almost identical in every unit test script indicates we should factor it out to a single place (DRY principle) and only for the TravisCI environment.

A separate method for creating a `virtualenv` for ones working environment for develop could leverage the same procedure, or be kept in our `CONTRIBUTION` documentation.